dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/datavalueset/DefaultDataValueSetService.java

```diff
@@ -1212,8 +1212,8 @@ private DataValue createDataValue( DataValueEntry dataValue, ImportContext conte
         internalValue.setAttributeOptionCombo( valueContext.getAttrOptionCombo() );
         internalValue.setValue( trimToNull( value ) );
         internalValue.setStoredBy( context.getStoredBy( dataValue ) );
-        internalValue.setCreated( dataValue.hasCreated() ? parseDate( dataValue.getCreated() ) : now );
-        internalValue.setLastUpdated( dataValue.hasLastUpdated() ? parseDate( dataValue.getLastUpdated() ) : now );
+        internalValue.setCreated( now );
```

Comment:
I understand that removing the `dataValue` as a source leaves us with just using `now` all the time but I wonder if that would then carry one the `now` for existing values. I checked the code and in `DefaultDataValueSetService#960` we are doing 
```java
context.getDataValueBatchHandler().updateObject( internalValue );
```
Shouldn't that mean the `created` of the existing value gets effectively overridden with the `now` of the import?

Reply:
missed that :)